### PR TITLE
Fix: M01

### PR DESCRIPTION
**Pull request by JonathanAmenechi**
## Types of changes

- [ ] Refactor/enhancement 
- [ ] Bug fix/behavior correction 
- [ ] New feature 
- [ ] Breaking change 
- [ ] Other, additional 

## Status

- [ ] Prefix PR title with `WIP` if necessary (changes not yet made).
- [ ] Add tests to cover changes as needed.
- [ ] Update documentation/changelog as needed.
- [ ] Verify all tests run correctly in CI and pass.
- [ ] Ready for review/merge.

### CODE REVIEW

File: src/UmaCtfAdapter.sol [hunk 1]
```diff
@@ -69,9 +69,16 @@ contract UmaCtfAdapter is IUmaCtfAdapter, Auth, BulletinBoard, IOptimisticReques
     /// Prepares the condition using the Adapter as the oracle and a fixed outcome slot count = 2.
     /// @param ancillaryData - Data used to resolve a question
     /// @param rewardToken   - ERC20 token address used for payment of rewards and fees
-    /// @param reward        - Reward offered to a successful proposer
-    /// @param proposalBond  - Bond required to be posted by OO proposers/disputers. If 0, the default OO bond is used.
-    /// @param liveness      - UMA liveness period in seconds. If 0, the default liveness period is used.
+    /// @param reward        - Reward offered to a successful OO proposer. 
+    ///                        Must be chosen carefully, to properly economically incentize OO proposers.
+    /// @param proposalBond  - Bond required to be posted by OO proposers/disputers.
+    ///                        If 0, the default OO bond is used.
```
Comment (L-Kov) [inline src/UmaCtfAdapter.sol:75]: can you specify the default in the comment here?

File: src/UmaCtfAdapter.sol [hunk 2]
```diff
@@ -69,9 +69,16 @@ contract UmaCtfAdapter is IUmaCtfAdapter, Auth, BulletinBoard, IOptimisticReques
...
+    ///                        Must be chosen carefully, to properly economically incentize OO proposers and disputers.
+    ///                        Questions expected to secure a large amount of value should consider a larger proposal bond. 
+    /// @param liveness      - OO liveness period in seconds. 
+    ///                        If 0, the default liveness period is used.
```
Comment (L-Kov) [inline src/UmaCtfAdapter.sol:79]: same here

File: src/UmaCtfAdapter.sol [hunk 3]
```diff
@@ -69,9 +69,16 @@ contract UmaCtfAdapter is IUmaCtfAdapter, Auth, BulletinBoard, IOptimisticReques
...
+    ///                        Must be chosen carefully, depending on the value backed by the question.
+    ///                        Questions expected to secure a large amount of value should consider a higher liveness period.
```
Comment (L-Kov) [inline src/UmaCtfAdapter.sol:81]: nit: higher -> longer

File: src/UmaCtfAdapter.sol (hunk 1 above)
Comment (JonathanAmenechi) [reply src/UmaCtfAdapter.sol:75]: The default is dependent on the reward token used, so chose not to

Comment (L-Kov) [reply src/UmaCtfAdapter.sol:75]: sg

**Review by L-Kov — APPROVED** [review]

**Review by mshrieve — APPROVED** [review]

**Pull request merged**